Firstapp/app/src/main/java/com/example/firstapp/FirstFragment.java

```diff
@@ -0,0 +1,60 @@
+package com.example.firstapp;
+
+import android.app.Notification;
+import android.os.Bundle;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.TextView;
+import android.widget.Toast;
+
+import androidx.annotation.NonNull;
+import androidx.fragment.app.Fragment;
+import androidx.navigation.NavDirections;
+import androidx.navigation.fragment.NavHostFragment;
+
+public class FirstFragment extends Fragment {
+
+    TextView countTextView;
+    @Override
+    public View onCreateView(
+            LayoutInflater inflater, ViewGroup container,
+            Bundle savedInstanceState
+    ) {
```

Comment:
The most common style is to put the closing parenthesis on the same line as the last argument.